### ISSUE TITLE
[FSIApp] tests are failing if Fluid is not compiled

### ISSUE THREAD

**Issue by philbucher** [KratosMultiphysics/Kratos#4102]
I get the following errors when FluidDynamics is not compiled:
~~~
======================================================================
ERROR: test_aitken_accelerator_variable_stiffness (convergence_accelerator_spring_test.ConvergenceAcceleratorSpringTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philippb/software/Kratos_prod/applications/FSIapplication/tests/convergence_accelerator_spring_test.py", line 104, in setUp
    self.model_part = self.ReadModelPart(GetFilePath("box_fluid"))
  File "/home/philippb/software/Kratos_prod/applications/FSIapplication/tests/convergence_accelerator_spring_test.py", line 93, in ReadModelPart
    model_part_io.ReadModelPart(model_part)
RuntimeError: Error: Element VMS3D is not registered in Kratos. Please check the spelling of the element name and see if the application which containing it, is registered corectly. [Line 84 ]



in kratos/sources/model_part_io.cpp:1422:void ModelPartIO::ReadElementsBlock(ModelPart&)
   kratos/sources/model_part_io.cpp:1458:void ModelPartIO::ReadElementsBlock(ModelPart&)
   kratos/sources/model_part_io.cpp:526:virtual void ModelPartIO::ReadModelPart(ModelPart&)
~~~

=> I think this can easily be solved by using elements from the core, I don't think you need the vms-elements for those tests

**Comment by rubenzorrilla** [comment]
Thanks for reporting. That's true, probably there is no necessity to use the fluid elements in such a simple test case.

**Comment by rubenzorrilla** [comment]
Fixed in #4104
